### PR TITLE
make main mode default

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/hosting/environment-variables/configuration-methods.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ export N8N_ENCRYPTION_KEY=<SOME RANDOM STRING>
 ### Execute all workflows in the same process
 
 !!! warning "Deprecated"
-	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 0.227.0. They will be removed in a future release. Main mode is now the default, so this step is not needed for version 0.227.0 and above.
+	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 0.227.0. They will be removed in a future release. Main mode is now the default, so this step isn't needed for version 0.227.0 and above.
```
Comment (ivov) [inline docs/hosting/environment-variables/configuration-methods.md:168]: Are we sure this is correct? From a quick check I'm not seeing this deprecation among the [0.227.0 commits](https://github.com/n8n-io/n8n/compare/n8n@0.226.2...n8n@0.227.0).

File: docs/hosting/environment-variables/environment-variables.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ Refer to [User management](/hosting/authentication/user-management-self-hosted/)
 
 | Variable | Type  | Default  | Description |
 | :------- | :---- | :------- | :---------- |
-| `EXECUTIONS_PROCESS` (**deprecated**) | Enum string: `main`, `own` | `own` | **Deprecated**. Whether n8n executions run in their own process or the main process. <br><br>Refer to [Execution modes and processes](/hosting/scaling/execution-modes-processes/) for more details. |
+| `EXECUTIONS_PROCESS` (**deprecated**) | Enum string: `main`, `own` | `main` | **Deprecated**. Whether n8n executions run in their own process or the main process. <br><br>Refer to [Execution modes and processes](/hosting/scaling/execution-modes-processes/) for more details. |
```
Comment (ivov) [inline docs/hosting/environment-variables/environment-variables.md:149]: Looks like [this URL](https://docs.n8n.io/hosting/scaling/execution-modes-processes/#own) should be updated as well?

image

Did not check if there are other places that might also need updating.

File: docs/hosting/environment-variables/configuration-methods.md (hunk 1 above)
Comment (StarfallProjects) [reply docs/hosting/environment-variables/configuration-methods.md:168]: I'm fairly sure that's when we added the note we were deprecating it . . .  I'll rephrase to be safe

File: docs/hosting/environment-variables/configuration-methods.md [hunk 3]
```diff
@@ -165,7 +165,7 @@ export N8N_ENCRYPTION_KEY=<SOME RANDOM STRING>
 ### Execute all workflows in the same process
 
 !!! warning "Deprecated"
-	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 0.227.0. They will be removed in a future release. Main mode is now the default, so this step is not needed for version 0.227.0 and above.
+	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 1.0. They will be removed in a future release. Main mode is now the default, so this step isn't needed for version 0.227.0 and above.
```
Comment (ivov) [inline docs/hosting/environment-variables/configuration-methods.md:168]:
```suggestion
	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 1.0. They will be removed in a future release. Main mode is now the default, so this step isn't needed for version 1.0 and above.
```

File: docs/hosting/scaling/execution-modes-processes.md [hunk 4]
```diff
@@ -13,9 +13,22 @@ Set the n8n mode using the following [environment variables](/hosting/environmen
...
 There are two options available when selecting how n8n handles execution processes: `own` and `main`.
 
+### Main
+
+This is the default mode. When using `main` mode, all executions will run in the main n8n process. This mode has the following benefits and disadvantages:
+
+| Benefits | Disadvantages |
+| :------- | :------------ |
+| **Minimal latency** | Can't take advantage of multiple CPUs. Single process can result in a bottleneck. |
+| **Resource efficiency**: only one CPU required. | Reduced stability, one crashed executions causes all others to fail. |
```
Comment (ivov) [inline docs/hosting/scaling/execution-modes-processes.md:23]:
```suggestion
| **Resource efficiency**: only one CPU required. | Reduced stability, one crashed execution causes all others to fail. |
```

File: docs/hosting/scaling/execution-modes-processes.md [hunk 5]
```diff
@@ -13,9 +13,22 @@ Set the n8n mode using the following [environment variables](/hosting/environmen
...
+
 ### Own
 
-This is the default setting in n8n. In `own` mode, each execution runs in its own, newly instantiated process (one execution equals one running process). Running in this mode has the following benefits and disadvantages:
+!!! warning "Deprecated"
+	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 1.0. They will be removed in a future release. Main mode is now the default, so this step isn't needed for version 0.227.0 and above.
```
Comment (ivov) [inline docs/hosting/scaling/execution-modes-processes.md:28]:
```suggestion
	n8n deprecated `own` mode and the `EXECUTIONS_PROCESS` flag in version 1.0. They will be removed in a future release. Main mode is now the default, so this step isn't needed for version 1.0 and above.
```

**Pull request merged**